From 31749cfa47babaf3a76292a40fabb768852dcc96 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jonas=20=C3=85dahl?= <jadahl@gmail.com>
Date: Thu, 5 Jun 2014 09:50:33 +0200
Subject: [PATCH] evdev: Follow libinput enum rename

s/libinput_pointer_button_state/libinput_button_state/
s/LIBINPUT_POINTER_BUTTON_STATE_/LIBINPUT_BUTTON_STATE_/

https://bugzilla.gnome.org/show_bug.cgi?id=731254
---
 clutter/evdev/clutter-device-manager-evdev.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/clutter/evdev/clutter-device-manager-evdev.c b/clutter/evdev/clutter-device-manager-evdev.c
index 3834042..1855438 100644
--- a/clutter/evdev/clutter-device-manager-evdev.c
+++ b/clutter/evdev/clutter-device-manager-evdev.c
@@ -1164,7 +1164,7 @@ process_device_event (ClutterDeviceManagerEvdev *manager_evdev,
         time = libinput_event_pointer_get_time (button_event);
         button = libinput_event_pointer_get_button (button_event);
         button_state = libinput_event_pointer_get_button_state (button_event) ==
-                       LIBINPUT_POINTER_BUTTON_STATE_PRESSED;
+                       LIBINPUT_BUTTON_STATE_PRESSED;
         notify_button (device, time, button, button_state);
 
         break;
-- 
2.1.0

